### PR TITLE
Coverity scan fixes

### PR DESCRIPTION
**Pull request by lpechacek**
Coverity has identified several instances of coding errors in QuickEvent code.  The most serious were uninitialized class members.  This patch set contains both plain fixes and code cleanup.  Please review.

https://scan.coverity.com/projects/quickbox

### CODE REVIEW

File: libqf/libqfqmlwidgets/src/tableview.cpp [hunk 1]
```diff
@@ -2067,12 +2067,12 @@ void TableView::removeSelectedRowsInline()
 
 bool TableView::edit(const QModelIndex& index, EditTrigger trigger, QEvent* event)
 {
-	qfLogFuncFrame() << "trigger:" << trigger << "event type:" << ((event)? event->type(): 0) << "editTriggers:" << editTriggers();
```
Comment (fvacek) [inline libqf/libqfqmlwidgets/src/tableview.cpp]: how do you know that event is always not NULL

**Comment by fvacek** [comment]
Good job, thanks a lot :)

File: libqf/libqfqmlwidgets/src/tableview.cpp (hunk 1 above)
Comment (lpechacek) [inline libqf/libqfqmlwidgets/src/tableview.cpp]: Good point.  Although Qt documentation doesn't say anything about event pointer value, grep through Qt sources shows that CurrentChanged trigger comes with NULL event pointer.  That means these checks must stay in place and event->accept() can be called without any checking as it's guarded by the trigger type test.  I'll drop this patch from the series.

**Comment by lpechacek** [comment]
Thanks for the review!  The defective patch has been dropped.

**Pull request merged**